tests/MongoDB.Driver.Examples/TransactionExamplesForDocs/WithTransactionExample1.cs

```diff
@@ -0,0 +1,100 @@
+
+            var connectionString = CoreTestConfiguration.ConnectionString.ToString();
+            DropCollections(
+                connectionString, 
+                CollectionNamespace.FromFullName("mydb1.foo"), 
+                CollectionNamespace.FromFullName("mydb2.bar"));
+            var result = string.Empty;
+
+            // Start Transactions withTxn API Example 1
+            // For a replica set, include the replica set name and a seedlist of the members in the URI string; e.g.
+            // string uri = "mongodb://mongodb0.example.com:27017,mongodb1.example.com:27017/?replicaSet=myRepl";
+            // For a sharded cluster, connect to the mongos instances; e.g.
+            // string uri = "mongodb://mongos0.example.com:27017,mongos1.example.com:27017:27017/";
+            var client = new MongoClient(connectionString);
+
+            // Prereq: Create collections. CRUD operations in transactions must be on existing collections.
+            var database1 = client.GetDatabase("mydb1");
+            var collection1 = database1.GetCollection<BsonDocument>("foo").WithWriteConcern(WriteConcern.WMajority);
+            collection1.InsertOne(new BsonDocument("abc", 0));
+
+            var database2 = client.GetDatabase("mydb2");
+            var collection2 = database2.GetCollection<BsonDocument>("bar").WithWriteConcern(WriteConcern.WMajority);
+            collection2.InsertOne(new BsonDocument("xyz", 0));
+
+            // Step 1: Start a client session.
+            using (var clientSession = client.StartSession())
+            {
+                // Step 2: Optional. Define options to use for the transaction.
+                var transactionOptions = new TransactionOptions(
+                    readPreference: ReadPreference.Primary,
+                    readConcern: ReadConcern.Local,
+                    writeConcern: WriteConcern.WMajority);
+
+                // Step 3: Define the sequence of operations to perform inside the transactions
+                result = clientSession.WithTransaction(
+                    (handle, token) =>
+                    {
+                        collection1.InsertOne(clientSession, new BsonDocument("abc", 1));
+                        collection2.InsertOne(clientSession, new BsonDocument("xyz", 999));
+                        return "Inserted into collections in different databases";
+                    },
+                    transactionOptions);
+            }
+            //End Transactions withTxn API Example 1
+
+            result.Should().Be("Inserted into collections in different databases");
+
+            var abcDocuments = collection1.Find(FilterDefinition<BsonDocument>.Empty).ToList();
+            abcDocuments.Count.Should().Be(2);
+            abcDocuments[0]["abc"].Should().Be(0);
+            abcDocuments[1]["abc"].Should().Be(1);
+
+            var xyzDocuments = collection2.Find(FilterDefinition<BsonDocument>.Empty).ToList();
```

Comment:
Rename variable to `collection2Documents`.

Reply:
done